java/impl/cmd-line/src/main/java/co/worklytics/psoxy/FileHandler.java

```diff
@@ -0,0 +1,104 @@
+package co.worklytics.psoxy;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.common.base.Preconditions;
+import lombok.NonNull;
+import lombok.SneakyThrows;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.inject.Inject;
+import java.io.File;
+import java.io.FileReader;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
+
+//@NoArgsConstructor(onConstructor_ = @Inject) //q: compile complaints - lombok annotation processing not reliable??
```

Comment:
if you have injects inside there is no need to add `(onConstructor_ = @Inject)`, it gets discovered automatically isn't it?

Reply:
it seems in dagger 2 you need both?? I get weird warnings